include/translated-term.php

```diff
@@ -299,4 +299,28 @@ protected function should_update_translation_group( $id, $translations ) {
 		$term = $this->get_object_term( $id, $this->tax_translations );
 		return empty( $term ) || ! empty( array_diff_assoc( $translations, $old_translations ) );
 	}
+
+	/**
+	 * Assigns a language to terms in mass.
+	 *
+	 * @since 1.2
+	 * @since 3.4 Moved from PLL_Admin_Model class.
+	 *
+	 * @param int[]               $ids  Array of post ids or term ids.
+	 * @param PLL_Language|string $lang Language to assign to the posts or terms.
```

Comment:
Same as above.

Reply:
Same question as above 🙂

Reply:
Done in 8b867bc1174b361bcae57de13b21591d2ad67e0c